src/test/java/org/wildfly/security/sasl/otp/OTPTest.java

```diff
@@ -902,6 +904,23 @@ public void testAuthenticationWithInvalidSequenceNumber() throws Exception {
     }
 
     @Test
+    public void testGeneratePasswordFromPassPhrase() throws Exception {
+        final String algorithm = ALGORITHM_OTP_SHA1;
+        final SaslClientFactory clientFactory = obtainSaslClientFactory(OTPSaslClientFactory.class);
```

Comment:
Just a minor note, this line can probably be removed since this test doesn't actually make use of the clientFactory.